Core/Inc/currentSense.h

```diff
@@ -0,0 +1,29 @@
+#ifndef INC_CURRENT_SENSE_H_
+#define INC_CURRENT_SENSE_H_
+
+/* ==================================================================== */
+/* ============================= INCLUDES ============================= */
+/* ==================================================================== */
+
+#include "bms.h"
+
+/* ==================================================================== */
+/* ============================= DEFINES ============================== */
+/* ==================================================================== */
+
+#define CURRENT_HIGH_RAIL_THRESHOLD             590//420
```

Comment:
Add some more documentation as to how these values were calculate - also list the sensor #